services/submission/src/test/java/app/coronawarn/server/services/submission/controller/RequestExecutor.java

```diff
@@ -65,25 +65,15 @@ public RequestExecutor(TestRestTemplate testRestTemplate) {
     return execute(HttpMethod.POST, new RequestEntity<>(body, headers, HttpMethod.POST, SUBMISSION_URL));
   }
 
-  public static HttpHeaders buildOkHeaders() {
-    HttpHeaders headers = setCwaAuthHeader(setContentTypeProtoBufHeader(new HttpHeaders()));
-
-    return setCwaFakeHeader(headers, "0");
-  }
-
-  public static HttpHeaders setContentTypeProtoBufHeader(HttpHeaders headers) {
-    headers.setContentType(MediaType.valueOf("application/x-protobuf"));
-    return headers;
-  }
-
-  public static HttpHeaders setCwaAuthHeader(HttpHeaders headers) {
-    headers.set("cwa-authorization", "TAN okTan");
-    return headers;
-  }
+  public ResponseEntity<Void> executePost(Collection<TemporaryExposureKey> keys) {
```

Comment:
I would prefer to call the overloaded method here other than having the same body with one different step. 

```java
return executePost(keys, HttpHeaderBuilder.builder()
        .contentTypeProtoBufHeader()
        .cwaAuthHeader()
        .cwaFakeHeader("0")
        .build());
```

Reply:
Could even extract the creation of the default header to something like `makeDefaultHeader` just to make it clearer.

Reply:
Ok, I changed that.